webui/src/lib/components/repository/changes.jsx

```diff
@@ -44,7 +44,7 @@ export const TreeItemRow = ({ entry, repo, reference, leftDiffRefID, rightDiffRe
     const [afterUpdated, setAfterUpdated] = useState(""); // state of pagination of the item's children
     const [resultsState, setResultsState] = useState({results:[], pagination:{}}); // current retrieved children of the item
     const [diffExpanded, setDiffExpanded] = useState(false); // state of a leaf item expansion
-    const enableDeltaDiff = JSON.parse(localStorage.getItem(`enable_delta_diff`));
+    const disableDeltaDiff = JSON.parse(localStorage.getItem(`disable_delta_diff`));
```

Comment:
1. Use single or double quotes
2. JSON.parse looks like overkill
```suggestion
    const disableDeltaDiff = localStorage.getItem('disable_delta_diff') === '1';
```